features/SideNavigation.feature

```diff
@@ -0,0 +1,165 @@
+@owner:lilla
+@author:lilla
+
+Feature: Navigating Sidenav
+    This allows users to easily navigate and select between articles and categories within our help center site
+
```

Comment:
Nice work on the scenario titles of your newer specs in here - most of the scenario titles here are working standalone